### PR TITLE
Wip: Add navbar

### PR DESCRIPTION
**Pull request by Andresws12**

### CODE REVIEW

**Comment by Andresws12** [comment]
[Add Navbar](https://trello.com/c/3ynia5w5/9-add-navbar)

**Comment by PVillasana** [comment]
> @PVillasana Could do you review my code and the interface?

@Andresws12 Yes i will

File: src/router/index.ts [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-        // this generates a separate chunk (about.[hash].js) for this route
-        // which is lazy-loaded when the route is visited.
-        component: () =>
-            import(/* webpackChunkName: "about" */ '../views/About.vue'),
+        component: About,
     },
 ];
 
 const router = new VueRouter({
+    mode: 'history',
```
Comment (PVillasana) [inline src/router/index.ts:24]: @Andresws12 What is the utility of mode: 'history' ?

File: src/store/main-store/MainGetters.ts [hunk 2]
```diff
@@ -3,6 +3,6 @@ import MainState from './MainState';
 
 export default class MainGetters extends Getters<MainState> {
     public get reversetestdata(): boolean {
```
Comment (PVillasana) [inline src/store/main-store/MainGetters.ts:5]: @Andresws12 Do this 'get' name has to be without camelCase?

File: src/components/navbar/TopNavbar.vue [hunk 3]
```diff
@@ -5,13 +5,17 @@
     >
         <template slot="brand">
             <top-navbar-item-link class="top-navbar_brand">
-                <b-icon
-                    class="top-navbar_left-menu"
-                    icon="bars"
-                    size="is-medium"
-                />
+                <mq-layout mq="lg+">
```
Comment (PVillasana) [inline src/components/navbar/TopNavbar.vue:8]: @Andresws12 This 'mq-layout'  will render the code inside if the media query pass the condition?

File: src/router/index.ts (hunk 1 above)
Comment (Andresws12) [reply src/router/index.ts:24]: With this propertie, the app don't use # in his URL, you can check the documentation here: https://router.vuejs.org/guide/essentials/history-mode.html#example-server-configurations

File: src/store/main-store/MainGetters.ts (hunk 2 above)
Comment (Andresws12) [reply src/store/main-store/MainGetters.ts:5]: @PVillasana Ups, my error..

File: src/components/navbar/TopNavbar.vue (hunk 3 above)
Comment (Andresws12) [reply src/components/navbar/TopNavbar.vue:8]: @PVillasana Yep, with this it's more easy do a differents layout to differents views

**Review by PVillasana — APPROVED** [review]

**Pull request merged**